hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/serdes/StringCodec.java

```diff
@@ -26,27 +26,27 @@
 public class StringCodec implements Codec<String> {
     @NonNull
     @Override
-    public String parse(@NonNull ReadableSequentialData input) throws IOException {
+    public String parse(final @NonNull ReadableSequentialData input) throws IOException {
         final var len = input.readInt();
         final var bytes = new byte[len];
         input.readBytes(bytes);
         return len == 0 ? "" : new String(bytes, StandardCharsets.UTF_8);
     }
 
     @Override
-    public void write(@NonNull String value, @NonNull WritableSequentialData output) throws IOException {
+    public void write(final @NonNull String value, final @NonNull WritableSequentialData output) throws IOException {
```

Comment:
in all the public methods null checks are missing

Reply:
Done